frontend/src/types.ts

```diff
@@ -4034,3 +4035,18 @@ export type OnboardingProduct = {
     url: string
     scene: Scene
 }
+
+export interface AnomalyCondition {
+    absolute_thershold: {
+        lower?: number
+        upper?: number
+    }
+}
+
+export interface AlertType {
+    id: number
+    name: string
+    insight?: number
+    target_value: string
+    anomaly_condition: AnomalyCondition
+}
```

Comment:
Most of the JSON we have around is using camelCase notations. Could we do the same here: `AlertType` should remain as is since it's python, but `AnomalyCondition` is stored as JSON, so could we use camelCase for the `absoluteThreshold` inside it.